docs/en/deploy/Running-Alluxio-On-Kubernetes.md

```diff
@@ -604,6 +633,26 @@ $ kubectl create -f alluxio-fuse-client.yaml
 If using the template, Alluxio is mounted at `/alluxio-fuse` and can be accessed via the POSIX-API
 across multiple containers.
 
+***Using `helm`***
+You can deploy the FUSE daemon by configuring the following properties:
+```properties
+fuse:
+  enabled: true
+  clientEnabled: true
+```
+
+Then follow the steps to install Alluxio with helm [here]({{ '/en/deploy/Running-Alluxio-On-Kubernetes.html#deploy-using-helm' | relativize_url }}).
+
+If Alluxio has already been deployed with helm and now you want to enable FUSE, you can manually add the FUSE daemons.
+```console
+# Generate YAML resources with helm template
+$ helm template --name alluxio helm-chart/alluxio/ --set fuse.enabled=true -x templates/fuse/daemonset.yaml -f ./config.yaml > "alluxio-fuse.yaml"
+$ helm template --name alluxio helm-chart/alluxio/ --set fuse.clientEnabled=true -x templates/fuse/client-daemonset.yaml -f ./config.yaml > "alluxio-fuse-client.yaml"
+# Manually apply the YAML files using kubectl
+$ kubectl create -f alluxio-fuse.yaml
```

Comment:
split the `kubectl`  commands into a separate block as only one of the `kubectl` or `helm` command is needed

Reply:
This is replaced by one `helm upgrade` command